lua/rainbow-delimiters/stack.lua

```diff
@@ -18,17 +18,21 @@
 local M = {}
 
 ---@class Stack
```

Comment:
We can also mark the class as "exact", meaning that there can be no additional fields. Do you agree?

```suggestion
---@class (exact) Stack
```

Reply:
Yes, we can do that. In principle we could probably do that with most classes in the type definitions, but for things that might change I think it makes sense to not include exact. Here Stack is stable though, so it makes a lot of sense to include it.